src/main/resources/swagger/cromwell.yaml

```diff
@@ -707,6 +714,11 @@ definitions:
       jobId:
         type: string
         description: Backend-specific job ID
+      failures:
+        type: object
+        schema:
+          $ref: '#/definitions/FailureMessage'
+        description: If the call failed, the set of recorded failure messages for the call.
```

Comment:
Speaking of I've noticed that Mr Proper (aka @mcovarr) tends to end in `.` in most places and Mr Slackadaisical (aka @geoffjentry) tends not to